llvm/test/MachineVerifier/test_g_incompatible_range.mir

```diff
@@ -1,4 +1,4 @@
-# RUN: not --crash llc -mtriple=amdgcn-mesa-mesa3d -mcpu=tahiti -run-pass=none %s -filetype=null 2>&1 | FileCheck %s
+# RUN: not --crash llc -o - -mtriple=amdgcn-mesa-mesa3d -mcpu=tahiti -run-pass=none %s 2>&1 | FileCheck %s
```

Comment:
This does not need -o and should use -filetype=null, you are only checking stderr

Reply:
e.g. in unsupported-subreg-index-aligned-vgpr-check.mir:
`# RUN: not --crash llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx942 -run-pass=none -filetype=null %s 2>&1 | FileCheck %s`

Reply:
sorry, I touched this line when experimenting, fixed, thank you!